tests/Functional/FactoryTest.php

```diff
@@ -34,8 +34,8 @@ public function many_to_one_relationship(): void
     {
         $categoryFactory = factory(Category::class, ['name' => 'foo']);
         $category = create(Category::class, ['name' => 'bar']);
-        $postA = create(Post::class, ['title' => 'title', 'body' => 'body', 'category' => $categoryFactory]);
-        $postB = create(Post::class, ['title' => 'title', 'body' => 'body', 'category' => $category]);
+        $postA = create(Post::class, ['type' => 'simple', 'title' => 'title', 'body' => 'body', 'category' => $categoryFactory]);
```

Comment:
This shouldn't be required. The discriminator column is internal to doctrine.

Reply:
Sure is, but I got a strange error in the tests. Gonna check that. I will update when I'm done with my changes and will state my changes :)